static/js/settings_org.js

```diff
@@ -862,10 +892,18 @@ export function register_save_discard_widget_handlers(container, patch_url) {
         e.preventDefault();
         e.stopPropagation();
         const save_button = $(e.currentTarget);
-        const subsection_id = save_button.attr("id").replace("org-submit-", "");
-        const subsection = subsection_id.split("-").join("_");
         const subsection_elem = save_button.closest(".org-subsection-parent");
 
+        if (for_realm_default_settings) {
+            const data = {
+                ...populate_data_for_request(subsection_elem),
+            };
+            save_organization_settings(data, save_button, patch_url);
+            return;
+        }
+
+        const subsection_id = save_button.attr("id").replace("org-submit-", "");
+        const subsection = subsection_id.split("-").join("_");
```

Comment:
Any suggestions to make this more readable or is it fine?

Reply:
This is kinda a complicated transformation.  In an ideal world, we'd figure out a way to not require this mapping, but I don't think we should block this change on doing so.

I'd write a comment above these couple lines explaining the background and giving an example.